src/core/Core.hpp

```diff
@@ -1,10 +1,24 @@
 #pragma once
+#include <string>
+#include <ctime>
 
 namespace bakcyl::core {
 
 class Core{
 public:
     Core();
+
+    struct Dataforemail {
+            std::string email_address_recipient;

```

Comment:
In src/sql/examples/Product.hpp camelCase has been used. Lets follow one convention.